dlite_entities_service/cli/_utils/global_settings.py

```diff
@@ -23,16 +28,19 @@ class ContextDict(TypedDict):
         """Global context for the CLI."""
 
         dotenv_path: Path
+        token: Token | None
 
 
 CONTEXT: ContextDict = {
-    "dotenv_path": (Path().cwd() / str(CONFIG.model_config["env_file"])).resolve()
+    "dotenv_path": (Path().cwd() / str(CONFIG.model_config["env_file"])).resolve(),
```

Comment:
This just seems like a such an ugly way to get what I presume to be a 'standard' file

Reply:
This ensures the env-file with the name defined in the general configuration model (`CONFIG`) is used as default from the directory in which the user is currently running the CLI.

So, this enables changing the default env-file name in the configuration model if so desired, without having to change this line.
And it enables the user to get the expected effect of using the env-file in the folder where one runs the CLI.